### ISSUE TITLE
sklern new versions generate problems

### ISSUE THREAD

**Issue by apascualgarcia** [omanor/MUSiCC#7]
Hi there,

Following the pip installation instructions fails because the function `cross_validation` seems too be deprecated (possibly from version 0.18 of [sklearn](https://scikit-learn.org/stable/whats_new/v0.18.html#model-selection-enhancements-and-api-changes)), returning the following error when the test is run:

```
Traceback (most recent call last):
  File "/home/apascual/usr/bin/anaconda3/envs/MUSiCC/bin/test_musicc.py", line 13, in <module>
    from musicc.core import correct_and_normalize
  File "/home/apascual/usr/bin/anaconda3/envs/MUSiCC/lib/python3.7/site-packages/musicc/core.py", line 21, in <module>
    from sklearn import cross_validation, linear_model  # for the LASSO
ImportError: cannot import name 'cross_validation' from 'sklearn' 
```

I would suggest to simply change the installation instructions to enforce the installation of an older version. I made it work with the following environment:

```
Name                    Version                   Build  Channel
ca-certificates           2019.1.23                     0  
certifi                   2019.3.9                 py27_0  
libedit                   3.1.20181209         hc058e9b_0  
libffi                    3.2.1                hd88cf55_4  
libgcc-ng                 8.2.0                hdf63c60_1  
libstdcxx-ng              8.2.0                hdf63c60_1  
MUSiCC                    1.0.2                     <pip>
ncurses                   6.1                  he6710b0_1  
numpy                     1.16.2                    <pip>
openssl                   1.1.1b               h7b6447c_1  
pandas                    0.24.2                    <pip>
pip                       19.0.3                    <pip>
pip                       19.0.3                   py27_0  
python                    2.7.16               h9bab390_0  
python-dateutil           2.8.0                     <pip>
pytz                      2018.9                    <pip>
readline                  7.0                  h7b6447c_5  
scikit-learn              0.15.2                    <pip>
scipy                     1.2.1                     <pip>
setuptools                40.8.0                   py27_0  
six                       1.12.0                    <pip>
sqlite                    3.27.2               h7b6447c_0  
tk                        8.6.8                hbc83047_0  
wheel                     0.33.1                   py27_0  
zlib                      1.2.11               h7b6447c_3 
```

This possibly explains issues #5 and #6 as well.

**Comment by jbholm** [comment]
Hi, 

I believe I am suffering from these same errors, as mention in #5 and #6 - 
Can you please provide specific instructions for how to install the specific environment you suggest above? (i.e. "simply change the installation instructions to enforce the installation"). 

Thanks!

**Comment by apascualgarcia** [comment]
Hi there,

As far as I remember, it should be something like this:

```
conda create --name MUSiCC python=2.7
source activate MUSiCC
pip install -U numpy scipy  pandas scikit-learn==0.15.2
pip install -U MUSiCC
```

Hope it helps.

**Comment by engal** [comment]
Hi,

This should be fixed as of MUSiCC version 1.0.3, which fixed deprecated scikit-learn usages.

Thanks for your interest in MUSiCC!
